app/models/dataset/WKRemoteDataStoreClient.scala

```diff
@@ -68,7 +68,7 @@ class WKRemoteDataStoreClient(dataStore: DataStore, rpc: RPC) extends LazyLoggin
   }
 
   def findPositionWithData(dataset: Dataset, dataLayerName: String): Fox[JsObject] =
-    rpc(s"${dataStore.url}/data/datasets/${dataset._id}/layers/$dataLayerName/findData")
+    rpc(s"${dataStore.url}/data/datasets/${dataset._organization}/${dataset.directoryName}/layers/$dataLayerName/findData")
```

Comment:
@frcroth thanks for adding the dataset upload test. This already saved me twice in this pr :see_no_evil: 

=> one place is here, where a request to the dataset was accidentally based on the dataset id and not on orga and the new directoryName